lib/cosmos/tools/cmd_tlm_server/cmd_tlm_server.rb

```diff
@@ -281,8 +284,9 @@ def limits_change_callback(packet, item, old_limits_state, value, log_change)
     #
     # @param event_type [Symbol] The type of limits event that occurred. Must
     #   be one of :LIMITS_SET which means the system limits set has changed,
-    #   :LIMITS_CHANGE which means an individual item has changed limits state, or
-    #   :LIMITS_SETTINGS which means an individual item has new settings
+    #   :LIMITS_CHANGE which means an individual item has changed limits state,
+    #   :LIMITS_SETTINGS which means an individual item has new settings, or
+    #   :STALE_PACKET which means a packet with limits has gone stale
```

Comment:
Hmm, this is a lie. It currently creates an event for ANY packet which goes stale. Do you think that's useful or should we only keep it to packets with limits?